pages/pokedex/[id].tsx

```diff
@@ -1,20 +1,62 @@
-import type { InferGetServerSidePropsType, NextPage } from "next";
+import { gql } from "@apollo/client";
+import type {
+  GetServerSideProps,
+  InferGetServerSidePropsType,
+  NextPage,
+} from "next";
+import { getApolloClient } from "../../apollo-client";
 import styles from "../../styles/Page.module.css";
+import { FeatureCard } from "../../components/organisms/FeatureCard";
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const { id } = context.query;
+
+  const client = getApolloClient();
+
+  const { data } = await client.query({
```

Comment:
Let's type the response for this client.query call, similar to how it's done here: https://github.com/brightsize/pokemon-apollo/blob/main/pages/index.tsx#L23

It might even be 1 to 1

Reply:
(as in make it _typeful_, not push the keyboard buttons like I tried lol)